Gemfile

```diff
@@ -16,6 +16,8 @@ gem "jbuilder", "~> 2.5"
 gem "bootsnap", ">= 1.1.0", require: false
 gem "bootstrap", "~> 4.0.0"
 gem "jquery-rails"
+gem "sorcery"
+gem "validates_email_format_of"
```

Comment:
I wouldn't use this gem, it's not very maintained. Consider using a regex instead:

https://guides.rubyonrails.org/active_record_validations.html#format

Reply:
I removed te gem and used regex. I used the gem because people were saying it was much simpler :)